### ISSUE TITLE
Add an option to hide or minimize __scene_inputs__ and __scene_outputs__ 

### ISSUE THREAD

**Issue by ZeroCool940711** [pedroCabrera/PyFlow#7]
I think we should be able to hide the __scene_inputs__ and __scene_outputs__ bars from the canvas, they use too much space and for people with small screen this could be a problem, also the color is too distracting, I think it would be better to have them hidden by default and have an option on the View menu to show them just like we have with the Variables Panel and other panels.

**Comment by pedroCabrera** [comment]
I agree the look is awful now, they are basically nodes, but don't they are not like other panels, they have pins and connect to other nodes, so they should be there,a option to hide them to be a small line with a button to expand its what I think is best, also they should no be affected by zooming in or out, now they are!
